src/pages/SignIn.jsx

```diff
@@ -0,0 +1,73 @@
+import {
+  AiFillTwitterCircle,
+  AiFillLinkedin,
+  AiFillGithub,
+} from 'react-icons/ai';
+import { IoLogoDiscord } from 'react-icons/io5';
+import SignInForm from '../components/SignIn/SignInForm';
+
+function SignIn() {
+  return (
+    <div className="flex sm:grid h-100 mb-16">
+      <div className="flex justify-center w-full sm:grid grid-cols-7 gap-2">
```

Comment:
The left part of the pages are common in the two pages. You maybe put them in a component and use them inside the `SignIn `and `SignUp` pages